asdf_astropy/converters/transform/tests/test_transform.py

```diff
@@ -0,0 +1,449 @@
+import asdf
+import pytest
+import numpy as np
+
+from astropy.modeling import models as astropy_models
+import astropy.modeling
+from astropy import units as u
+from numpy.testing import assert_array_equal
+from asdf.tests.helpers import assert_roundtrip_tree, yaml_to_asdf
+
+
+# TODO: Uncomment this fixture once astropy converters
+# have been fully implemented.
+# @pytest.fixture(autouse=True)
+# def remove_astropy_extensions():
+#     """
+#     Disable the old astropy extension so that it doesn't
+#     confuse our test results.
+#     """
+#     with asdf.config_context() as config:
+#         config.remove_extension(package="astropy")
+#         yield
+
+
+def assert_model_roundtrip(model, tmpdir, version=None):
```

Comment:
Is this different from the existing `assert_roundtrip_tree` assertion helper in `asdf`?  I see you import that above, but do not use it anywhere here.  I think it does make more sense to compare models than just the trees, as the model-to-tree conversion needs to be tested as well.

Reply:
It is different, the `Converter` API doesn't include that `assert_equals` function so `assert_roundtrip_tree` will compare the models with `__eq__` and fail.  I think we'll still be able provide a useful test helper but until we figure that out I just implemented a model-specific one here.

Reply:
Ah, that makes a lot of sense.  Carry on.  🚀